packages/openactive-integration-tests/test/helpers/request-helper.js

```diff
@@ -41,23 +62,52 @@ class RequestHelper {
     return response;
   }
 
-  async get(stage, url, headers) {
-    return await this._request(stage, 'GET', url, null, headers);
```

Comment:
This would call chakram.get(url, null, headers). If you look at the docs (http://dareid.github.io/chakram/jsdoc/module-chakram.html#.get), you can see that chakram.get only takes 2 args - therefore, "headers" was being ignored

Reply:
Good catch, that was quite a hack previously!